sdk/go/common/resource/plugin/context_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/opentracing/opentracing-go/mocktracer"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/testing/diagtest"
+	"github.com/stretchr/testify/require"
+)
+
+func TestContextRequest_race(t *testing.T) {
+	// Create a new mock tracer
+	tracer := mocktracer.New()
+
+	// Create a new root span
+	rootSpan := tracer.StartSpan("root")
```

Comment:
Let's drop the two trivial comments.
The effort it takes to read the two comments is the same as the effort it takes to read those two lines of code, so they don't add anything here.

```suggestion
	tracer := mocktracer.New()
	rootSpan := tracer.StartSpan("root")
```

(I suggest checking out [this blog post](http://antirez.com/news/124) by the author of Redis. It provides a nice way to categorize different kinds of code comments and the reasoning for why what he calls "trivial comments" should be avoided.)